doap_HUDI.rdf

```diff
@@ -41,6 +41,11 @@
         <created>2019-10-24</created>
         <revision>0.5.0</revision>
       </Version>
+      <Version>
+        <name>Apache Hudi-incubating 0.5.0</name>
```

Comment:
hello @smarthi, it is `Apache Hudi-incubating 0.5.1`.

Reply:
Damn!!1 yeah u r right - PR again - my bad.